scripts/automator/build/gcc-defaults

```diff
@@ -4,20 +4,24 @@ cc="${ccache}gcc${postfix}"
 cxx="${ccache}g++${postfix}"
 ld="gcc${postfix}"
 ranlib="gcc-ranlib${postfix}"
+cflags+=(-fdiagnostics-color=auto)
+
+# ccache can cache GCC's preprocessor pass
+if [[ -n "${ccache}" ]]; then 
+	cflags+=(-fpch-preprocess)
+fi
 
 # Release-type additions
 TYPES+=(release debug warnmore pgotrain fdotrain optinfo
         asan uasan usan tsan)
 
-cflags+=(-fdiagnostics-color=auto)
-
 cflags_debug+=("${cflags[@]}" -g -fstack-protector -fno-omit-frame-pointer)
 cxxonly_debug+=("${cxxonly[@]}" -Weffc++)
 
 # Note: associative-math is needed for vectorization of floating point
 #       calculations, which also relies on no-signed-zeros and
 #       no-trapping-math.
-cflags_release+=("${cflags[@]}" -DNDEBUG -O3 -fstrict-aliasing
+cflags_release+=("${cflags[@]}" -DNDEBUG -O3 -ftree-vectorize -fstrict-aliasing
```

Comment:
`-ftree-vectorize` is  already included in `-O3`

Reply:
Thanks for catching that @dreamer; Will drop this commit in a couple hours. (Or, If you want to merge and drop this commit - that works too!)

Reply:
Thanks @dreamer ; dropped this!